truss/templates/control/control/application.py

```diff
@@ -36,8 +47,29 @@ def handle_error(exc):
             return exc
 
         app.logger.exception(exc)
+        if isinstance(
```

Comment:
In the future, you might consider avoiding `isinstance` as a pattern, as its out of line with polymorphism best practices.  For example, could this code be reworked to look like this: 
```
try:
  raise exc
except HttpException:
  return exc
except (
                UnsupportedPatch,
                PatchFailedRecoverable,
                PatchFailedUnrecoverable,
                InadmissiblePatch,
            ):
  error_type = _camel_to_snake_case(type(exc).__name__)
            return {
                "error": {
                    "type": error_type,
                    "msg": str(exc),
                }
            }

...
```

Another option is to map these classes into a handler registry and lookup the handler for that specific exception (I happen to like the above a bit more from the cleanliness standpoint).

A good thought exercise is if you find yourself using `isinstance` or any reflection, consider if there is a way to use the abstracted methods in a generic way without converting to the concrete type.